.circleci/config.yml

```
@@ -0,0 +1,70 @@
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.4.6
+
+workflows:
+  test:
+    jobs:
+      - test_chrome
+      - test_firefox
+
+jobs:
+  test_chrome:
+    docker:
+      - image: cimg/node:10.24.0-browsers
```

```suggestion
      - image: cimg/node:10.24.1-browsers
```
In Travis.ci we used a newer version of Node. To be consistent let's update it.